src/main/java/org/jabref/gui/ClipBoardManager.java

```diff
@@ -67,16 +80,44 @@ public String getContents() {
         return result;
     }
 
+    // Get the text from Primary
```

Comment:
It's java convention to use `/** ... */` for these comments. Search for "JavaDoc" if you want to learn more (and see also which tags are supported).